package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sauce-testcafe-runner",
-  "version": "0.0.0",
+  "version": "0.3.0",
```

Comment:
We can leave this as `0.0.0`, it's the release pipeline that should bump it up like in the cypress runner repo.
The cypress pipeline bumps it, but doesn't commit the changes (which is fine!), so that the resulting artifact has the correct version without having the need to modify the source.

Reply:
@alexplischke I did not think about that. I need to check how `release-it` (underlying tool) is behaving when there is a 0.0.0 version. 


Personally, I'll prefer to have the real version committed (at list in the tag), so by checkout-ing that said tag, you are 100% identical to the source used. So to build an artifact, you just have to checkout + build, thus artifact building steps just have to actually build it, not modify the source. 

Perfect example is `saucetpl.tgz`, update of the version will not be anymire responsibility of artifact build.

Reply:
Aye, either way is fine (as long as it's the same process used across all our image repos) 💪

Reply:
@alexplischke  To keep consistent with previous behaviour, `package(-lock)?.json` will contains `0.0.0` and it will be updated before bundling (either `saucetpl`, `windows-runner`, `macos-runner`.
It will also avoid to create noise by adding some commits (just like currently).

I'll replicate this to the other runners, but this can be reviewed as of now.